index.html

```diff
@@ -1280,6 +1270,158 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `*_localized` members
+        </h3>
+        <p>
+          A <dfn>localizable member</dfn> is a [=manifest=] member that can be
+          localized. Each [=localizable member=] of the [=manifest=] has a
+          corresponding <dfn>localized member</dfn> with the suffix
+          `_localized`. For example, the [=manifest/name=] member is a
+          [=localizable member=], where `name_localized` is the corresponding
+          [=localized member=].
+        </p>
+        <p>
+          [=Localized members=] contain a [=localization map=] that defines
+          [=localized values=] for different locales denoted by a [=language
+          tag=]. If present and if the [=localization map=] contains a
+          [=localized value=] that matches the user's localization settings,
+          the user agent SHOULD override the value of the [=localizable
+          member=].
+        </p>
+        <aside class="example" title=
+        "Overriding the application name with localized values">
+          <pre class="json">
+            {
+              "lang": "en-US",
+              "name": "Color Picker",
+              "name_localized": {
+                "en-GB": "Colour Picker",
+                "fr": "Sélecteur de Couleur"
+              }
+            }
+          </pre>
+        </aside>
+        <p>
+          The <dfn>localization map</dfn> is an [=ordered map=] whose key is a
+          [=language tag=] and whose value is a [=localized value=]. The
+          <dfn>localized value</dfn> represents the value applied for the
+          [=language tag=] specified as the key. The value matches the type of
+          its corresponding [=localizable member=]. For example, a [=localized
+          value=] for the [=manifest/name=] member could directly be a
+          [=string=].
+        </p>
+        <p class="note">
+          To support multilingual content and ensure proper display and
+          accessibility, it is possible to specify a different language tag and
+          text direction for a localized value. This is needed for situations
+          where a term or text must be presented in a language different from
+          the user's set language.
```

Comment:
This covers the language tag not matching the language map key case (which is rare). However, the far more common requirement is that the direction of the translation does not match the default direction of the application. For example, every right-to-left value will need this when the default language of the application is a left-to-right language (or vice versa).

As a result, I would leave out "text direction" in the note and mention direction separately.